spec/imsc-hrm.html

```diff
@@ -684,9 +844,12 @@ <h2>Paint Text</h2>
         </tbody>
       </table>
 
-      <p class='note'>The choice of font by the presentation processor can increase rendering complexity. For instance, a cursive
-      font can generally result in a given character yielding different typographical glyphs depending on context, even if latin
-      script is used.</p>
+      <p class='note'>The choice of font by the <dfn
+      data-cite="ttml2#terms-presentation processor">presentation
+      processor</dfn> can increase rendering complexity. For instance, a cursive
+      font can generally result in a given character yielding different
+      typographical glyphs depending on context, even if latin script is
+      used.</p>
```

Comment:
This text was modified since this branch was made and will conflict - I suggest taking the current version following the merge of #42.